source/contracts/trading/Order.sol

```diff
@@ -53,6 +53,7 @@ library Order {
     // Constructor
     //
 
+    // No validation is needed here as it is simply a librarty function for organizing data
```

Comment:
While this is true if we only look one layer deep, is it still true that all functions that take an `Order.Data` properly validate the `.market` property?  My guess is not, which means we should do validation here.

Reply:
Its a library so even if we wanted to we could not. My point is that because this is a library it is effectively not an external contract call

Reply:
More on this, any caller would also be passing in the used market, so they should have done validation